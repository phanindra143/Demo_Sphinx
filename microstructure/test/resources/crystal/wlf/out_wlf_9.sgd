*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=    89.8857
  !k_ij(1,2)=     1.2174
  !k_ij(2,2)=    90.0379
  !k_ij(1,3)=    -1.0736
  !k_ij(2,3)=     0.4010
  !k_ij(3,3)=    84.9631
MP,Kxx,1,   89.88571
MP,Kyy,1,   90.03793
MP,Kzz,1,   84.96306
MP,Kxx,2,   90.03793
MP,Kyy,2,   89.88571
MP,Kzz,2,   84.96306
MP,Kxx,3,   90.03793
MP,Kyy,3,   89.88571
MP,Kzz,3,   84.96306
MP,Kxx,4,   89.88571
MP,Kyy,4,   90.03793
MP,Kzz,4,   84.96306
MP,Kxx,5,   89.88571
MP,Kyy,5,   90.03793
MP,Kzz,5,   84.96306
MP,Kxx,6,   90.03793
MP,Kyy,6,   89.88571
MP,Kzz,6,   84.96306
MP,Kxx,7,   90.03793
MP,Kyy,7,   89.88571
MP,Kzz,7,   84.96306
MP,Kxx,8,   89.88571
MP,Kyy,8,   90.03793
MP,Kzz,8,   84.96306
MP,Kxx,9,   84.96306
MP,Kyy,9,   89.88571
MP,Kzz,9,   90.03793
MP,Kxx,10,   89.88571
MP,Kyy,10,   84.96306
MP,Kzz,10,   90.03793
MP,Kxx,11,   89.88571
MP,Kyy,11,   84.96306
MP,Kzz,11,   90.03793
MP,Kxx,12,   84.96306
MP,Kyy,12,   89.88571
MP,Kzz,12,   90.03793
MP,Kxx,13,   84.96306
MP,Kyy,13,   89.88571
MP,Kzz,13,   90.03793
MP,Kxx,14,   89.88571
MP,Kyy,14,   84.96306
MP,Kzz,14,   90.03793
MP,Kxx,15,   89.88571
MP,Kyy,15,   84.96306
MP,Kzz,15,   90.03793
MP,Kxx,16,   84.96306
MP,Kyy,16,   89.88571
MP,Kzz,16,   90.03793
MP,Kxx,17,   90.03793
MP,Kyy,17,   84.96306
MP,Kzz,17,   89.88571
MP,Kxx,18,   84.96306
MP,Kyy,18,   90.03793
MP,Kzz,18,   89.88571
MP,Kxx,19,   84.96306
MP,Kyy,19,   90.03793
MP,Kzz,19,   89.88571
MP,Kxx,20,   90.03793
MP,Kyy,20,   84.96306
MP,Kzz,20,   89.88571
MP,Kxx,21,   90.03793
MP,Kyy,21,   84.96306
MP,Kzz,21,   89.88571
MP,Kxx,22,   84.96306
MP,Kyy,22,   90.03793
MP,Kzz,22,   89.88571
MP,Kxx,23,   84.96306
MP,Kyy,23,   90.03793
MP,Kzz,23,   89.88571
MP,Kxx,24,   90.03793
MP,Kyy,24,   84.96306
MP,Kzz,24,   89.88571
*endif
