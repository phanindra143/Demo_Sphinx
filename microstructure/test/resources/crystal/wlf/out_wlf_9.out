            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\13
             3.00000      -1404014.07347          4360.87550         12264.78574
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000          -280.42710            86.33096
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000          2159.18357      -1406633.14766         38321.17669
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000          -222.53793         15625.00000          -474.73156
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         19399.33362         58803.77645      -1327168.23389
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -47.63601          -771.30068         15625.00000
  /%
